index.bs

```diff
@@ -4231,6 +4231,7 @@ promises returned by its <a lt="writable stream writer">writer</a>'s {{WritableS
     return new WritableStream({
       start(controller) {
         ws.onerror = () => controller.error(new Error("The WebSocket errored!"));
+        ws.onclose = () => controller.error(new Error("The server closed the connection unexpectedly!"));
```

Comment:
Just to check my understanding, the server could close it non-abruptly, right? In which case close would fire but not error.